src/containers/mentor-home-page/experience/Experience.js

```diff
@@ -0,0 +1,48 @@
+import { useTranslation } from 'react-i18next'
+import { Box, Typography } from '@mui/material'
+
+import AppTextField from '~/components/app-text-field/AppTextField'
+
+import img from '~/assets/img/mentor-home-page/become-tutor/experience.png'
+import { styles } from './experience.styles'
+
+const Experience = ({ data, handleChange, handleBlur, errors, btnsBox }) => {
```

Comment:
To handle errors of the step, pass a callback function that should be called with a boolean if the step has errors.